packages/docs/community/release-process/smart-contracts.md

```diff
@@ -119,18 +131,21 @@ celocli governance:show --proposalID <proposalId> --jsonTransactions "upgrade_pr
 
 ### Verify Proposed Release
 
-Verify that the proposed upgrade activates contract addresses which match compiled bytecode from the tagged release candidate exactly. Include initialization data from the CGP to verify via (add `-i initialization_data.json` in that case)
+This script serves the same purpose as `verify-deployed` but for a not-yet
+accepted contract upgrade (in the form of the proposal.json you fetched in the step prior). It gives you the confidence that the branch specified in the `-b` flag in (same as `check-versions`) will be the resulting network state of the proposal if executed. It does so by going over all Celo Core Contracts and determining updates to the Registry pointers, proxy or implementation contracts and verifying their implied bytecode against the compiled source code.
```

Comment:
was `proposal.json` "fetched" or "generated"? If fetched from previous step, the json files have two different names in this doc which is a little confusing.